src/ui/windows/TogglDesktop/TogglDesktop/ui/controls/TimelineRunningTimeEntryBlock.xaml.cs

```diff
@@ -31,6 +32,26 @@ private void OnThumbTopDragDelta(object sender, DragDeltaEventArgs e)
         private void OnThumbTopDragCompleted(object sender, DragCompletedEventArgs e)
         {
             ViewModel.ChangeStartTime();
+            ViewModel.IsDragged = false;
+        }
+
+        private void OnThumbDragStarted(object sender, DragStartedEventArgs e)
+        {
+            ViewModel.IsDragged = true;
+        }
+
+        private TimelineTimeEntryBlockPopup _popup;
+        protected override void OnMouseEnter(MouseEventArgs e)
+        {
+            _popup ??= new TimelineTimeEntryBlockPopup();
+            _popup.DataContext = ViewModel;
+            _popup.Popup.PlacementTarget = this;

```

Comment:
Why did you decide to change the logic of the popup positioning for the running time entry?
Old issues like this one are back now.
![image](https://user-images.githubusercontent.com/16451391/99375486-76698b80-28cc-11eb-8a55-7aed2abfdcab.png)